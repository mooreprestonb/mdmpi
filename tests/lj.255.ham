# istep ham ke pe dof=765
0 -1.03672 0.380505 -1.41723
1 -1.03692 0.381487 -1.41841
2 -1.03705 0.379684 -1.41673
3 -1.03707 0.375651 -1.41272
4 -1.03708 0.370657 -1.40773
5 -1.03716 0.366994 -1.40415
6 -1.03718 0.366739 -1.40391
7 -1.03718 0.369907 -1.40708
8 -1.03714 0.374489 -1.41162
9 -1.03737 0.378201 -1.41557
10 -1.03738 0.380224 -1.41761
11 -1.03726 0.380424 -1.41769
12 -1.03694 0.37847 -1.41541
13 -1.03667 0.374618 -1.41129
14 -1.03647 0.370322 -1.40679
15 -1.03651 0.367577 -1.40409
16 -1.03633 0.367426 -1.40375
17 -1.03639 0.368506 -1.40489
18 -1.03636 0.368285 -1.40464
19 -1.0362 0.366631 -1.40284
20 -1.03624 0.365972 -1.40222
21 -1.0365 0.367039 -1.40354
22 -1.03657 0.367477 -1.40405
23 -1.03659 0.36563 -1.40222
24 -1.03616 0.363188 -1.39934
25 -1.03607 0.363218 -1.39929
26 -1.03591 0.36636 -1.40227
27 -1.03599 0.370334 -1.40633
28 -1.036 0.372652 -1.40865
29 -1.03597 0.372159 -1.40813
30 -1.03592 0.36919 -1.40511
31 -1.03585 0.365378 -1.40123
32 -1.03616 0.362444 -1.3986
33 -1.0362 0.361 -1.3972
34 -1.0363 0.360954 -1.39726
35 -1.03653 0.362106 -1.39864
36 -1.03652 0.363967 -1.40049
37 -1.03653 0.365656 -1.40218
38 -1.03669 0.366153 -1.40284
39 -1.03664 0.364711 -1.40135
40 -1.03647 0.361294 -1.39777
41 -1.03638 0.356824 -1.3932
42 -1.03623 0.352924 -1.38915
43 -1.03616 0.351378 -1.38754
44 -1.03598 0.353475 -1.38946
45 -1.03581 0.359167 -1.39498
46 -1.03602 0.366779 -1.4028
47 -1.03618 0.373767 -1.40995
48 -1.03611 0.377774 -1.41388
49 -1.03607 0.377541 -1.41361
50 -1.03587 0.373453 -1.40932
51 -1.03544 0.367142 -1.40258
52 -1.03549 0.360245 -1.39573
53 -1.03533 0.353572 -1.3889
54 -1.03506 0.347975 -1.38304
55 -1.03527 0.345474 -1.38074
56 -1.03519 0.347852 -1.38304
57 -1.03547 0.35414 -1.38961
58 -1.03561 0.361002 -1.39661
59 -1.03557 0.365619 -1.40119
60 -1.03568 0.36743 -1.40311
61 -1.0356 0.367623 -1.40323
62 -1.03534 0.367868 -1.40321
63 -1.03507 0.369188 -1.40425
64 -1.03526 0.371616 -1.40688
65 -1.0353 0.374767 -1.41007
66 -1.0354 0.378223 -1.41363
67 -1.03539 0.381229 -1.41662
68 -1.03551 0.382741 -1.41825
69 -1.03542 0.382104 -1.41753
70 -1.03571 0.379353 -1.41507
71 -1.03581 0.375037 -1.41085
72 -1.03554 0.370133 -1.40567
73 -1.03543 0.366025 -1.40145
74 -1.03544 0.364101 -1.39955
75 -1.03551 0.365055 -1.40057
76 -1.03539 0.368415 -1.40381
77 -1.0355 0.372726 -1.40822
78 -1.03557 0.376204 -1.41178
79 -1.0356 0.37733 -1.41293
80 -1.03594 0.375466 -1.4114
81 -1.036 0.371279 -1.40728
82 -1.03607 0.366192 -1.40226
83 -1.03619 0.361417 -1.39761
84 -1.03614 0.358026 -1.39417
85 -1.03632 0.357064 -1.39338
86 -1.03638 0.358675 -1.39506
87 -1.03644 0.361921 -1.39836
88 -1.03642 0.365794 -1.40221
89 -1.03647 0.369734 -1.40621
90 -1.03639 0.373284 -1.40967
91 -1.03649 0.375412 -1.4119
92 -1.03638 0.375048 -1.41142
93 -1.03621 0.373072 -1.40928
94 -1.03631 0.372662 -1.40897
95 -1.03637 0.375976 -1.41235
96 -1.03686 0.381551 -1.41841
97 -1.03705 0.386649 -1.4237
98 -1.03739 0.389748 -1.42713
99 -1.03732 0.390177 -1.4275
