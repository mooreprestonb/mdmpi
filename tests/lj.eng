# istep d=765, ham ke pe psr plr lj elec e_short e_long e_corr 1/d
0 -1.03401 0.311226 -1.34523 -1.34523 -4.40133e-06 -1.34523 -4.08911e-06 3.12211e-07 -4.40133e-06 0 0.00130719
10 -1.03469 0.308835 -1.34352 -1.34352 -4.29152e-06 -1.34352 -3.95875e-06 3.32774e-07 -4.29152e-06 0 0.00130719
20 -1.03536 0.320617 -1.35598 -1.35597 -4.23142e-06 -1.35597 -3.98661e-06 2.44808e-07 -4.23142e-06 0 0.00130719
30 -1.03458 0.312855 -1.34743 -1.34743 -3.89202e-06 -1.34743 -3.74608e-06 1.45948e-07 -3.89202e-06 0 0.00130719
40 -1.03481 0.315024 -1.34984 -1.34983 -3.5602e-06 -1.34983 -3.31347e-06 2.4673e-07 -3.5602e-06 0 0.00130719
50 -1.03521 0.319449 -1.35466 -1.35466 -3.41335e-06 -1.35466 -3.11037e-06 3.02987e-07 -3.41335e-06 0 0.00130719
60 -1.03473 0.318099 -1.35283 -1.35282 -3.82252e-06 -1.35282 -3.70696e-06 1.15557e-07 -3.82252e-06 0 0.00130719
70 -1.03479 0.316008 -1.3508 -1.35079 -4.37878e-06 -1.35079 -4.47388e-06 -9.50983e-08 -4.37878e-06 0 0.00130719
80 -1.0353 0.318703 -1.354 -1.35399 -4.63506e-06 -1.35399 -4.67845e-06 -4.33923e-08 -4.63506e-06 0 0.00130719
90 -1.03689 0.33966 -1.37655 -1.37654 -4.56956e-06 -1.37654 -4.44398e-06 1.25583e-07 -4.56956e-06 0 0.00130719
100 -1.03655 0.327593 -1.36414 -1.36414 -4.39298e-06 -1.36414 -4.10082e-06 2.92156e-07 -4.39298e-06 0 0.00130719
110 -1.0363 0.331204 -1.3675 -1.3675 -4.31485e-06 -1.3675 -4.14907e-06 1.65778e-07 -4.31485e-06 0 0.00130719
120 -1.03707 0.349726 -1.38679 -1.38679 -3.99755e-06 -1.38679 -3.80309e-06 1.94461e-07 -3.99755e-06 0 0.00130719
130 -1.03648 0.335748 -1.37223 -1.37222 -3.68228e-06 -1.37222 -3.57964e-06 1.02639e-07 -3.68228e-06 0 0.00130719
140 -1.0373 0.326041 -1.36334 -1.36333 -3.37086e-06 -1.36333 -3.32506e-06 4.58028e-08 -3.37086e-06 0 0.00130719
150 -1.03729 0.331469 -1.36876 -1.36876 -2.78234e-06 -1.36876 -2.5111e-06 2.71239e-07 -2.78234e-06 0 0.00130719
160 -1.03742 0.332265 -1.36969 -1.36969 -2.30582e-06 -1.36969 -1.80146e-06 5.04358e-07 -2.30582e-06 0 0.00130719
170 -1.03785 0.329313 -1.36716 -1.36716 -1.97522e-06 -1.36716 -1.47567e-06 4.99548e-07 -1.97522e-06 0 0.00130719
180 -1.03751 0.343908 -1.38142 -1.38142 -1.49929e-06 -1.38142 -9.47065e-07 5.52222e-07 -1.49929e-06 0 0.00130719
190 -1.03716 0.339032 -1.37619 -1.37619 -1.35278e-06 -1.37619 -7.34634e-07 6.18151e-07 -1.35278e-06 0 0.00130719
200 -1.03753 0.328286 -1.36581 -1.36581 -1.93481e-06 -1.36581 -1.49548e-06 4.39337e-07 -1.93481e-06 0 0.00130719
210 -1.03776 0.334842 -1.3726 -1.3726 -2.28891e-06 -1.3726 -2.05256e-06 2.36353e-07 -2.28891e-06 0 0.00130719
220 -1.03832 0.350996 -1.38932 -1.38931 -2.71238e-06 -1.38931 -2.55677e-06 1.5561e-07 -2.71238e-06 0 0.00130719
230 -1.03849 0.345429 -1.38392 -1.38392 -3.62776e-06 -1.38392 -3.77615e-06 -1.48387e-07 -3.62776e-06 0 0.00130719
240 -1.03853 0.35026 -1.38879 -1.38878 -4.07396e-06 -1.38878 -4.2453e-06 -1.71343e-07 -4.07396e-06 0 0.00130719
250 -1.03851 0.344403 -1.38291 -1.38291 -4.09954e-06 -1.38291 -4.38746e-06 -2.87918e-07 -4.09954e-06 0 0.00130719
260 -1.03848 0.347057 -1.38554 -1.38553 -4.08977e-06 -1.38553 -4.3567e-06 -2.66921e-07 -4.08977e-06 0 0.00130719
270 -1.03856 0.361052 -1.39961 -1.39961 -3.81341e-06 -1.39961 -4.03243e-06 -2.1902e-07 -3.81341e-06 0 0.00130719
280 -1.03866 0.369223 -1.40789 -1.40788 -3.33242e-06 -1.40788 -3.28621e-06 4.62129e-08 -3.33242e-06 0 0.00130719
290 -1.03787 0.335369 -1.37324 -1.37323 -3.0075e-06 -1.37323 -2.88694e-06 1.20554e-07 -3.0075e-06 0 0.00130719
300 -1.0381 0.353058 -1.39115 -1.39115 -2.95406e-06 -1.39115 -2.79252e-06 1.61543e-07 -2.95406e-06 0 0.00130719
310 -1.03794 0.349437 -1.38737 -1.38737 -3.0077e-06 -1.38737 -3.2107e-06 -2.02997e-07 -3.0077e-06 0 0.00130719
320 -1.03765 0.331978 -1.36963 -1.36962 -2.84682e-06 -1.36962 -3.04448e-06 -1.97659e-07 -2.84682e-06 0 0.00130719
330 -1.03643 0.354801 -1.39124 -1.39123 -3.10612e-06 -1.39123 -3.34342e-06 -2.37299e-07 -3.10612e-06 0 0.00130719
340 -1.03736 0.350533 -1.3879 -1.38789 -3.41694e-06 -1.38789 -3.86507e-06 -4.48129e-07 -3.41694e-06 0 0.00130719
350 -1.03728 0.333654 -1.37093 -1.37093 -3.35898e-06 -1.37093 -3.87165e-06 -5.12678e-07 -3.35898e-06 0 0.00130719
360 -1.03749 0.35314 -1.39064 -1.39063 -3.5813e-06 -1.39063 -4.24952e-06 -6.68221e-07 -3.5813e-06 0 0.00130719
370 -1.03736 0.341776 -1.37914 -1.37914 -3.45874e-06 -1.37914 -4.14221e-06 -6.83474e-07 -3.45874e-06 0 0.00130719
380 -1.03708 0.335219 -1.3723 -1.3723 -3.02846e-06 -1.3723 -3.39112e-06 -3.62658e-07 -3.02846e-06 0 0.00130719
390 -1.03662 0.330703 -1.36733 -1.36732 -2.69772e-06 -1.36732 -2.74349e-06 -4.57637e-08 -2.69772e-06 0 0.00130719
400 -1.03612 0.343726 -1.37985 -1.37985 -2.63178e-06 -1.37985 -2.77066e-06 -1.38885e-07 -2.63178e-06 0 0.00130719
