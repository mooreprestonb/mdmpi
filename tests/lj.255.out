Running ../src/lj on METHANE.CMM.UPENN.EDU (id=24458)
# nmol=255, ntime=100, dt=0.01, xbox=6.70307, ipot=3, perd=3, ncell=5
# rest="lj.255.rest" ham="lj.255.ham" conf="lj.255.conf"
Verlist2 being used.
Getting initial energy and force
Initial values = h= -1158.12, ke=289.022, pe=-1447.14
Starting MD ntime = 100 with 765 dof
0 -1158.32 289.83 -1448.15
10 -1158.58 287.571 -1446.15
20 -1157.67 279.496 -1437.17
30 -1158.03 278.084 -1436.12
40 -1157.65 276.03 -1433.68
50 -1157.69 278.305 -1435.99
60 -1158.08 282.45 -1440.53
70 -1158.45 274.02 -1432.47
80 -1158.23 278.389 -1436.62
90 -1157.98 271.926 -1429.91
Writing restart file lj.255.rest.100
